internal/controller/alertmanager_controller.go

```diff
@@ -46,6 +46,7 @@ func SetupAlertmanagerReconciler(mgr ctrl.Manager, conf config.Config) error {
 
 	// Setup the controller
 	return ctrl.NewControllerManagedBy(mgr).
+		Named("alertmanager").
```

Comment:
Adding this otherwise the controller shows up as "secret" in the logs

![image](https://github.com/user-attachments/assets/b38bbcbb-577a-4076-b4d0-7f78b26478f0)